fe1-web/src/features/home/screens/Launch.tsx

```diff
@@ -39,19 +40,19 @@ const Launch: FunctionComponent<IPropTypes> = ({ navigation }) => {
     }
 
     getNetworkManager().connect(inputAddress);
+
     createLao(laoName)
-      .then((channel: Channel) =>
+      .then((channel: Channel) => {
         subscribeToChannel(channel).then(() => {
+          // after subscribing to the LAO channel, add the server address to the lao state
+          dispatch(addLaoServerAddress(getLaoIdFromChannel(channel), inputAddress));
```

Comment:
`dispatch` is being called in an async context here. This could be a problem.
Should this be moved into a thunk ?

Reply:
Yeah that sounds like an idea! What issues could this cause?

Reply:
Actually I think you're right, we might not need that.

If we were to create a thunk, it'd allow us to do something like `dispatch(createNewLao(laoName))` (which would do all the calls in the background), but then the thunk would have to handle everything: lao creation, subscription, etc.

In our case, there's no side effect associated to the state change, we can happily leave the code as-is.